hacks/isolateVersion/2-deleteOtherVersions.sh

```diff
@@ -0,0 +1,29 @@
+function delete_version() {
+  notify "Deleting version $1..."
+  folder=versioned_docs/$VERSION
+  sidebars=versioned_sidebars/$VERSION-sidebars.json
+  rm -rf $folder
+  rm $sidebars
+  git add versioned_docs versioned_sidebars
+  git commit -m "archiving: delete version $VERSION"
+}
+
+function delete_next() {
+  notify "Deleting next version..."
+  folder=docs
+  sidebars=sidebars.js
```

Comment:
this also needs to delete the "next" Optimize docs -- via the `optimize` folder and `optimize_sidebars.js` file! I'll add that when I do my next version.